### ISSUE TITLE
Responses are bilingual-ish or French/bilingual sometimes

### ISSUE THREAD

**Issue by anikbrazeau** [cds-snc/platform-forms-client#3398]
## Description
Inconsistent language

## Steps to reproduce
1. Go to Responses
2. Download >5
3. Open receipt from FR interface
4. See how parts of the file are FR only and some are bilingual

## Details
- **Browser**: Edge
- **Operating system**: Windows
- **Environment**: App router PR
- **Language**: French
- **Viewport**: tablet

## Expected behaviour
All French or all bilingual?

## Screenshots or videos

![bilingualish2](https://github.com/cds-snc/platform-forms-client/assets/38330843/362ce013-a5d3-45b3-adbd-46710d45b95d)
![bilingualish](https://github.com/cds-snc/platform-forms-client/assets/38330843/bf3b7072-88f7-4ded-9721-2af7dc5f7eb4)

**Comment by Abi-Nada** [comment]
De-scoping from App Router work, as this persists in Staging/Prod (but isn't a severe/critical bug that stops a client from creating/publishing/managing a form).

**Comment by connorscarolyns** [comment]
The questions must be presented in bilingual language so that responses can be assessed by the language of submission. This is a requirement, not a bug. The rest of the text in the response should be presented in the language of the interface. So if I'm working in French, all the "interface" language in the response should be in French, and the questions in the table should be bilingual, French first. 

In short, the screenshot is functioning as expected.

**Comment by Abi-Nada** [comment]
Closing since functionality is behaving as expected
